local/bin/py/build/configurations/pull_config_preview.yaml

```
@@ -323,6 +323,29 @@
           dependencies: ["https://github.com/DataDog/browser-sdk/blob/master/packages/rum/README.md"]
           title: RUM Browser Monitoring
           kind: documentation
+    - action: pull-and-push-file
+      branch: ruth/docs-1257
```

```suggestion
      branch: master
```
Change right before merge